filters/sharpen.go

```diff
@@ -0,0 +1,81 @@
+package filters
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/zalando/skipper/filters"
+	"gopkg.in/h2non/bimg.v1"
+)
+
+// For infomations about the parameters meanings and default values have a look here:
+// http://www.vips.ecs.soton.ac.uk/supported/current/doc/html/libvips/libvips-convolution.html#vips-sharpen
+
+const (
+	SharpenName = "sharpen"
+)
+
+type sharpen bimg.Sharpen
```

Comment:
I think you should define your own struct here, and not use the existing one from `bimg`. We are not in control on how the struct from `bimg` looks like and in case they change it in a future version of the library, all of our code would break.